test/functional/includeconf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
It'd be nice to have a couple of other subtests:

- including `--includeconf` in the command line argument has no effect
- including `includeconf` in an includeconf file has no effect.